src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.FileOperations.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        internal const uint SEM_FAILCRITICALERRORS = 1;
```

Comment:
The CoreRT version that has it next to SetErrorMode is better - this constant only make sense together with the SetErrorMode API